plugins/Marketplace/vue/src/ManageLicenseKey/ManageLicenseKey.vue

```diff
@@ -0,0 +1,164 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock
+    :content-title="translate('Marketplace_LicenseKey')"
+    class="manage-license-key"
+  >
+    <div class="manage-license-key-intro" v-html="$sanitize(manageLicenseKeyIntro)"></div>
+
+    <div class="manage-license-key-input">
+      <Field
+        uicontrol="text"
+        name="license_key"
+        v-model="licenseKey"
+        :placeholder="licenseKeyPlaceholder"
+        full-width="true"
```

Comment:
```suggestion
        :full-width="true"
```

That should fix the console message I found:

> [Vue warn]: Invalid prop: type check failed for prop "fullWidth". Expected Boolean, got String with value "true".